### PR TITLE
~new features~ and added config optional arg to chicago-business

### PR DESCRIPTION
**Pull request by murray-cecile**
Major changes are as follows:

In feature_generation.py:
1. I changed the count by ZIP to count by census tract. (The census tract codes should be in merged_business_govt_data.csv as of last week, but let me know if you don't have them and I'll zip my joined_licenses version.)
2. I made a handful of basic text filters on the business license description, like "food", "hair", "vehicle" etc

In chicago-business.py
3. I added an optional --config argument to chicago-business.py so that you can specify a different config file path if you want. This allowed me to create multiple config files for each train/test split and then run a shell script (run-models.sh) that's basically just `time python chicago-business.py --config <config>` over and over again.

### CODE REVIEW

File: code/chicago-business.py [hunk 1]
```diff
@@ -111,21 +106,50 @@ def get_pipeline(config_path):
...
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help = "config file path", default = "config.yml")
+    args = parser.parse_args()
+
+    config, script_dir = get_config(args.config)
+    # pipeline = get_pipeline(config)
+    # pipeline.run()
+            
+    pipeline = (get_pipeline(config, script_dir)
```
Comment (satejsoman) [inline code/chicago-business.py]: do you need to change this back to pipeline.run() and re-enable model saving?

File: code/get_baseline_failure.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import configparser
```
Comment (satejsoman) [inline code/get_baseline_failure.py]: feel free to rename chicago-business to chicago_business so you can import things from it, this is going to get out of sync very quickly.

File: code/chicago-business.py (hunk 1 above)
Comment (murray-cecile) [reply code/chicago-business.py]: yep good catch

File: code/get_baseline_failure.py (hunk 2 above)
Comment (murray-cecile) [reply code/get_baseline_failure.py]: will do!

**Comment by murray-cecile** [comment]
merging in new features + tract aggregation + --config arg

**Pull request merged**